sentence_transformers/backend.py

```diff
@@ -180,32 +182,126 @@ def export_dynamic_quantized_onnx_model(
     if file_suffix is None:
         file_suffix = f"{quantization_config.weights_dtype.name.lower()}_quantized"
 
-    save_or_push_to_hub_onnx_model(
+    save_or_push_to_hub_model(
         export_function=lambda save_dir: quantizer.quantize(quantization_config, save_dir, file_suffix=file_suffix),
         export_function_name="export_dynamic_quantized_onnx_model",
         config=quantization_config,
         model_name_or_path=model_name_or_path,
         push_to_hub=push_to_hub,
         create_pr=create_pr,
         file_suffix=file_suffix,
+        backend="onnx",
     )
 
 
-def save_or_push_to_hub_onnx_model(
+def export_static_quantized_openvino_model(
+    model: SentenceTransformer,
+    quantization_config: OVQuantizationConfig,
```

Comment:
```suggestion
    quantization_config: OVQuantizationConfig | dict | None,
```
Looks like `OVConfig` also accepts a dict, and perhaps we can just initialize `OVQuantizationConfig` without any arguments if it's `None`? Then users don't have to manually write the relatively complex import `from optimum.intel import OVQuantizationConfig` if they don't want to update any defaults.